src/rpc/mining.cpp

```diff
@@ -870,6 +870,134 @@ UniValue estimatesmartfee(const JSONRPCRequest& request)
 /* ************************************************************************** */
 /* Merge mining.  */
 
+/* The variables below are used to keep track of created and not yet
+       submitted auxpow blocks.  Lock them to be sure even for multiple
```

Comment:
Please fix the whitespace here and in the line below.  (So that all lines start on a column with "The".)